### DISCUSSION TITLE
Differing results

### DISCUSSION THREAD

**Discussion by kathmcp** [phoebe-project/phoebe2#518] [Questions]
Hi again,

I have completed a run using Emcee solver using 2 filters, however when I ask for the results using "uncertainties_from_distribution_collection" and when I ask for them individually, they are different. What is the reason for this? The program took nearly 3 days to run, so I would like to be confident I am doing it right before running it again!
![DiffResults2](https://user-images.githubusercontent.com/81473857/131217021-e9b8fff9-b021-489e-aa33-943997d6332a.PNG)

Thanks!

**Comment by kecnry** [comment]
Looking back into the source code, it looks like there might be a discrepancy in phoebe of using mean when adopting the face-values but using percentiles (so median for the central value) when presenting the uncertainties from the distribution.  We will have to discuss whether or not to make a change to that in a future release or bugfix, but I do think it likely explains the different numbers you're seeing here and you can choose which you want to use manually in the meantime.

Can you also just confirm that `adopt_values@emcee_sol` is set to `True`?  If `adopt_values=False`, then only the distributions themselves would be adopted and the parameter face-values would not change... which would also explain a difference between the solution and values in `context='component'` after adopting.

**Reply by kecnry** [reply]
We've decided to make this change in the [2.3.53 bugfix release](https://github.com/phoebe-project/phoebe2/releases/tag/2.3.53).  You should be able to save your bundle before adopting the solution, update phoebe, load the bundle, and then when you adopt it will use the median value and should be in agreement with the values you see from the uncertainties.  Thanks for bringing this to our attention!

**Reply by kathmcp** [reply]
Ok fantastic! Thanks again Kyle!
